changelog.yaml

```diff
@@ -0,0 +1,33 @@
+# THE CHANGELOG FORMAT IS BASED ON https://github.wdf.sap.corp/ConCISe/Concepts/blob/master/docs/Implementation/cloudci_backend_release_process/changelog.yaml

```

Comment:
Reference to sap-internal resource -> remove!!!

Reply:
Done